model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -81,6 +88,17 @@ def allowed_keys(cls):
 
         return cls.perf_analyzer_args + cls.input_to_options + cls.input_to_verbose
 
+    @classmethod
+    def additive_keys(cls):
+        """
+        Returns
+        -------
+        list of str
+            The keys, within allowed_keys, that are additive
+        """
+
+        return cls.additive_args[:]
```

Comment:
Why return a copy?

Reply:
To match the behavior of `allowed_keys()` above, because if the list returned by `allowed_keys()` is modified, the underlying `perf_analyzer_args`, `input_to_options` and `input_to_verbose` are not modified.
If a copy is not returned, modifying the return will modify the underlying `additive_args` that might not be desirable.